plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/network/rest/wpcom/wc/tracker/TrackerStore.kt

```diff
@@ -7,7 +7,11 @@ import javax.inject.Singleton
 
 @Singleton
 class TrackerStore @Inject internal constructor(private val restClient: TrackerRestClient) {
-    suspend fun sendTelemetry(appVersion: String, site: SiteModel): WooPayload<Unit> {
-        return restClient.sendTelemetry(appVersion, site)
+    suspend fun sendTelemetry(
+        appVersion: String,
+        site: SiteModel,
+        installationDateIso8601: String? = null
```

Comment:
I wouldn't introduce a default argument here - there's no case when we want to invoke `sendTelemetry` method without `installationDateIso8601` argument

```suggestion
        installationDateIso8601: String?
```